src/main/java/seedu/address/logic/parser/AddLeaveCommandParser.java

```diff
@@ -32,13 +34,17 @@ public AddLeaveCommand parse(String args) throws ParseException {
 
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_LEAVE_TITLE, PREFIX_LEAVE_DATE_START,
-                        PREFIX_LEAVE_DATE_END, PREFIX_LEAVE_DESCRIPTION);
+                        PREFIX_LEAVE_DATE_END, PREFIX_LEAVE_DESCRIPTION, PREFIX_LEAVE_STATUS);
```

Comment:
I am not sure if we need to add that in our UG, but I think some people may be confused if they should put status because it should be one of the properties of "leave." And there is a bug report who deliberately tested this.....

Reply:
understood, i saw that issue afterwards. i think the stuff i push should be able to help you handle that case + other unexpected prefixes :)

Reply:
i'm not sure which push you are referring to....

Reply:
OK in the event I'm not allowed to push my PR, I think the fix for yours is to wrap the Description in a try-catch block (see #116), since it is the description's validation that is failing and not being caught that results in a bug. Specifically detecting for the presence of the status prefix will not address the general case, as we would encounter the same bug with a command like `add-leave 1 title/... start/... end/... d/some description some-invalid-prefix/some text`, or even `add-leave 1 title/... start/... end/... d/$$$ aka invalid characters`

Reply:
oh wait i understand the problem, i will fix it soon, previously I could not reproduce the bug....